tangelo/toolboxes/ansatz_generator/_qubit_cc.py

```diff
@@ -130,8 +130,8 @@ def get_idxs_deriv(qham_term, *qham_qmf_data):
             gen = (idx, "Y") if idxs == "" else (idx, "X")
             idxs = idxs + f" {idx}" if idxs != "" else f"{idx}"
             gen_tup += (gen, )
-    # Generators must have at least two flip indices
-    if len(gen_tup) > 1:
+    # Generators must have at least one flip index
+    if len(gen_tup) > 0:
```

Comment:
Naïve question: is this increasing significantly the pool size, i.e. increase the time to get all the generators per iteration?

Reply:
No, it only seems to apply for the JKMN mapping and it is not a significant increase. At most it is N extra generators where N is the number of qubits.

Reply:
Alright! It is fine